doc/sphinx-guides/source/developers/intro.rst

```diff
@@ -55,4 +55,4 @@ As a developer, you also may be interested in these projects related to Datavers
 
 ----
 
-Next: :doc:`dev-environment`
+Next: :doc:`windows`
```

Comment:
Shouldn't this require also a change of _previous_ link in ```windows```?

Reply:
I thought so too but windows doc doesn't have next/before links. On the other hand maybe I should just fix windows doc in this pr.